app/models/update_certificate_hub_use_confirmation_event.rb

```diff
@@ -0,0 +1,8 @@
+class UpdateCertificateHubUseConfirmationEvent < AggregatedEvent
+  belongs_to_aggregate :certificate
+  data_attributes :hub_use_confirmation_at
+
+  def attributes_to_apply
+    { hub_use_confirmation_at: hub_use_confirmation_at }
```

Comment:
```suggestion
    { hub_use_confirmation_at: Time.now }
```
or something along this line? We shouldn't need to worry about what stuff to pass in.